### PR TITLE
Restaurant 

### PR DESCRIPTION
**Pull request by YinkTech**
This project is a javascript webpack and es6 functionalities. with different types of module pages.

### CODE REVIEW

File: src/pageload.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import leadImage from './assets/pexels-cassiano-psomas-1045199.jpg';
+import firstImage from './assets/pexels-dhyamis-kleber-3584317.jpg';
+import fifthImage from './assets/pexels-tookapic-7782.jpg';
+import thirdImage from './assets/pexels-chevanon-photography-312418.jpg';
+
+export function pageload() {
```
Comment (soieducation) [inline src/pageload.js]: - [x] As this project introdues ES6, all functions should be [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

File: src/dishes.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    const dishesContainer = document.createElement("div");
+    dishesContainer.classList.add("dishescont");
+
+    const firstDish = document.createElement("div");
+    firstDish.classList.add("firstDish");
+    const firstImg = document.createElement("img");
+    firstImg.classList.add("first-dish-imag");
+    firstImg.src = firstImge;
+    firstDish.textContent = "Black Tea as low as \n $2";
```
Comment (soieducation) [inline src/dishes.js:22]: - [x] It'll be good if the picture sizes are compressed to allow them to load faster

Comment (soieducation) [reply src/dishes.js:22]: - [x] The pictures are not currently aligned, kindly adjust them
Screenshot (384)

Screenshot (385)

File: src/pageload.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    fifthDish.textContent = "Stake with Chips as low as \n $7";
+    fifthImg.alt = "fifth dish Image";
+    
+    fifthDish.append(fifthImg);
+    dishesContainer.append(fifthDish);
+
+    const thirdDish = document.createElement("div");
+    thirdDish.classList.add("firstDish");
+    const thirdImg = document.createElement("img");
+    thirdImg.classList.add("first-dish-imag");
```
Comment (soieducation) [inline src/pageload.js:58]: - [x] The pictures here should be aligned also 
image

**Review by soieducation — CHANGES_REQUESTED** [review]
Hi @YinkTech, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/dishes.js (hunk 2 above)
Comment (YinkTech) [reply src/dishes.js:22]: this how it displaying at my end
Screenshot (27)

Screenshot (28)

Comment (soieducation) [reply src/dishes.js:22]: Noted

File: src/pageload.js (hunk 3 above)
Comment (soieducation) [reply src/pageload.js:58]: Noted

**Comment by soieducation** [comment]
Kindly implement the rest of the comments above

File: src/pageload.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import firstImage from './assets/pexels-dhyamis-kleber-3584317.jpg';
 import fifthImage from './assets/pexels-tookapic-7782.jpg';
 import thirdImage from './assets/pexels-chevanon-photography-312418.jpg';
 
-export function pageload() {
+export let pageload =  () => {
```
Comment (soieducation) [inline src/pageload.js]: - [ ] All arrow functions should be declared using `const` not `let` as they are not going to be reassigned, kindly modify them before merging.

**Review by soieducation — APPROVED** [review]
Hi @YinkTech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**